CHANGELOG.md

```diff
@@ -48,398 +48,331 @@ This document tracks pending changes to packages. It is facilitating the write-u
-  `asEvent` implementations were removed as well as redundant `HasImplementation` instances)
+- Make use of the `requires` keyword to enforce the interface hierarchy (in particular the `asDisclosure` and
+  `asEvent` implementations were removed)
 
-- Let `Election` and `ElectionEffect` implement the `Disclosure` interface.
-
-- Add check that instruments exist for `Distribution` and `Replacement`
+- The `Distribution` and `Replacement` lifecycle rules check that the target and procued instruments are active
 
 ### Daml.Finance.Settlement
 
-- In the settlement `Factory`, the id values used for the `Instruction`s were modified to accurately
-  reflect their order within the `Batch`.
+- Update of SDK version and dependencies
+
+- In the settlement `Factory`, the id values used for the `Instruction`s were modified to accurately reflect their order within the `Batch`.
 
 - In the `Batch`, the order of the `settledCids` were changed to match the initial order of the
   instructions in the batch.
 
-- Fix for `Instruction` and `IntermediatedStatic`, replaced `groupOn` by `sortAndGroupOn`.
-
-- Dependencies update
+- Bug fix: replace `groupOn` by `sortAndGroupOn` in the `Instruction` and `IntermediatedStatic` templates
 
-- Style changes
+- Lock pledged holding when allocating to an `Instruction`: the pledged holding is locked to the instruction's requestors and the outgoing controllers of the sending account
 
-- Added locking to the `Instruction` (pledge is locked to requestors and the outgoing controllers of
-  the sending account)
+- Make use of the `requires` keyword to enforce the interface hierarchy (in particular the `asDisclosure`
+  implementation was removed)
 
-- Makes use of `requires` to enforce the interface hierarchy (in particular the `asDisclosure`
-  implementation was removed as well as redundant `HasImplementation` instances)
+- When an `Allocation` (resp. `Approval`) takes place, a check has been added to ensure that either the
+  `sender` or the `custodian` (resp. the `receiver` or the `custodian`) is among the choice authorizers
 
-- When an `Allocation` (`Approval`) takes place, a check has been added to ensure that either the
-  `sender` or the `custodian` (`receiver` or the `custodian`) is among the authorizers.
+- When reallocation (resp. re-approval) occurs, it is required that the `signedSenders`
+  (resp. `signedReceivers`) of the `Instruction` are part of the authorizing set
 
-- When reallocation (re-approval) occurs, it is required that the `signedSenders`
-  (`signedReceivers`) of the `Instruction` are part of the authorizing set.
+- Add additional checks to the pass-through allocation/approval process. Specifically, verify that the specified pass-through `Instruction` is actually part of the `Batch`. These checks detect settlement failures during the allocation/approval stage rather than waiting until settlement occurs.
```

Comment:
```suggestion
- Add additional checks to the pass-through allocation/approval process. Specifically, verify that 
the specified pass-through `Instruction` is actually part of the `Batch`. These checks detect 
settlement failures during the allocation/approval stage rather than waiting until settlement 
occurs.
```